tika/server.go

```diff
@@ -39,6 +40,41 @@ type Server struct {
 	url  string // url is derived from port.
 	port string
 	cmd  *exec.Cmd
+	child *ChildOptions
+}
+
+// Command line parameters that can be used when tika is run with the -spawnChild option.
```

Comment:
```suggestion
// ChildOptions represent command line parameters that can be used when Tika is run with the -spawnChild option.
```

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences (and capitalizing Tika).